src/test/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionSupportSpec.scala

```diff
@@ -182,6 +184,275 @@ class DataUseRestrictionSupportSpec extends FreeSpec with Matchers with DataUseR
       }
 
     }
+
+    "ORSP-based restrictions" - {
+      val ontologyDAO = new MockOntologyDAO
+
+      "when translating DUOS restrictions to FireCloud restrictions" - {
+        "should return no attributes if DUOS is empty" in {
+          val attrs = generateStructuredUseRestrictionAttribute(new DuosDataUse, ontologyDAO)
+          assert(attrs.isEmpty)
+        }
+        "should translate boolean TRUEs" in {
+          val attrs = generateStructuredUseRestrictionAttribute(new DuosDataUse(
+            generalUse = Some(true),
+            hmbResearch = Some(true),
+            commercialUse = Some(true),
+            vulnerablePopulations = Some(true),
```

Comment:
See comments above about this field - I don't think we should be using it to model the pediatric case. Similar to other usages in this test.